test-prow-e2e.sh

```diff
@@ -17,7 +17,7 @@ trap copyArtifacts EXIT
 
 # don't log kubeadmin-password
 set +x
-BRIDGE_KUBEADMIN_PASSWORD="$(cat "${INSTALLER_DIR}/auth/kubeadmin-password")"
+BRIDGE_KUBEADMIN_PASSWORD="$(cat "${KUBEADMIN_PASSWORD_FILE:-${INSTALLER_DIR}/auth/kubeadmin-password}")"
```

Comment:
Why is this change neccesary?  Where is `KUBEADMIN_PASSWORD_FILE` set?

Reply:
Somehow it got added, Now I removed it, Could you please review it now